admin/settings/class-admin-apple-settings-section-api.php

```diff
@@ -85,10 +90,12 @@ public function __construct( $page ) {
 	 */
 	public function get_section_info() {
 		return sprintf(
-			'%s <a target="_blank" href="https://developer.apple.com/news-publisher/">%s</a> %s.',
-			__( 'Enter your Apple News credentials below. See', 'apple-news' ),
-			__( 'the Apple News documentation', 'apple-news' ),
-			__( 'for detailed information', 'apple-news' )
+			'%s <a target="_blank" href="https://developer.apple.com/documentation/apple_news/apple_news_api/getting_ready_to_publish_and_manage_your_articles">%s</a> %s <a target="_blank" href="https://support.apple.com/guide/news-publisher/use-your-cms-with-news-publisher-apd88c8447e6/icloud">%s</a> %s.',
+			__( 'Please upload your Apple News channel configuration file below. Please see', 'apple-news' ),
+			__( 'the Apple News API documentation', 'apple-news' ),
+			__( 'and', 'apple-news' ),
+			__( 'the News Publisher documentation', 'apple-news' ),
+			__( 'for detailed information. For further assistance, please contact your Apple News representative.', 'apple-news' )
```

Comment:
I know this is the same format as it was previously, but really, it should be inverted, like this:
```suggestion
			__( 'Please upload your Apple News channel configuration file below. Please see %sthe Apple News API documentation%s and %sthe News Publisher documentation%s for detailed information. For further assistance, please contact your Apple News representative.', 'apple-news' ),
			'<a target="_blank" href="https://developer.apple.com/documentation/apple_news/apple_news_api/getting_ready_to_publish_and_manage_your_articles">',
			'</a>',
			'<a target="_blank" href="https://support.apple.com/guide/news-publisher/use-your-cms-with-news-publisher-apd88c8447e6/icloud">',
			'</a>',
```